source/runbooks/dr-process.html.md.erb

```diff
@@ -0,0 +1,362 @@
+|Core-Logging|Creates the core logging account resources |
+|Core-Security|Creates the core security account resources |
+|Cloud-Platform-Transit-Gateways|Across all regions. |
+|Core-VPC-*|Creates the core VPC resources in the VPC accounts, needed after the core-networking-services. |
+| Modernisation-Platform | Creates key resources such as S3 state buckets for the platform |
+
+
+### Test Accounts
+
+
+If there is limited availability, power off all infrastructure in the test accounts. All are useless until the platform is running.
+
+
+
+## Single Account
+
+Things to remember
+
+- Dont forget the statefile, this could cause issues when rebuilding and generating plans.
+
+
+### Will it be returning
+
+
+If this is a test account, waiting may be the easier idea. If AWS can promise us this is a temporary issue with an estimated fix time, we can pass this to the team and agree our next step.
+
+
+### Rebuild account
+
+
+Assuming this is a production account with no time given to return, please follow the steps.
+
+
+#### Workflows
+
+
+Rather than rebuilding an account through workflows, I would run steps manually to easier debug issues, however, technically, you could start to run the account creation process with our github workflows.
+
+**Member Accounts** - Located [here](https://github.com/ministryofjustice/modernisation-platform/actions/workflows/new-environment.yml), running the new environment workflow should begin the steps of an account creation for member accounts.
+
+**Core Accounts** - Located [here](https://github.com/ministryofjustice/modernisation-platform/actions), on the left you will see the deployment workflows for all core based accounts, required for the building on the platform.
+
+However, I would run manually just to monitor the plan, help troubleshoot issues, as sometimes these scenarios are not as cut and dry as all infrastructure has vanished from one account.
+
+
+#### Manually
+
+
+1.) Run a terraform plan against the production workspace, in modernisation-platform .
+
+2.) All changes should appear, using your admin credentials, run an apply.
+
+3.) Assuming the apply runs as planned, login to the account and check that vpcs, etc look correct.
+
+4.) In modernisation-platform-environments , run terraform against the correct account.
+
+
+
+### Things to remember
+
+
+- Statefile for the lost account will exist, this might need to be removed.
+- Any backups on the account or volumes will be lost.
+
+
+## Disaster Recovery Running outside of London.
+
+
+If the London infrastructure is to be down for the foreseeable future, running from another region is the next step.
+
+
+
+### Informing teams
+
+
+Firstly, inform the following channels of the situation and put forward a contactable member of staff from our team.
+
+
+`#ask-modernisation-platform`
+
+
+`#modernisation-platform`
+
+
+`#modernisation-platform-alerts`
```

Comment:
this channel does not exist and we should also post it in the `#modernisation-platform-updates` channel

Reply:
this is still not addressed